app/exposureHistory.ts

```diff
@@ -0,0 +1,37 @@
+import dayjs from 'dayjs';
+
+export type Posix = number;
+
+export interface Possible {
+  kind: 'Possible';
+  date: Posix;
+  duration: number;
+  totalRiskScore: number;
+  transmissionRiskLevel: number;
+}
+
+export interface NoKnown {
+  kind: 'NoKnown';
+  date: Posix;
+}
+
+export type ExposureDatum = Possible | NoKnown;
+
+export type ExposureHistory = ExposureDatum[];
+
+export const blankHistory = (): NoKnown[] => {
+  const now = Date.now();
+  const daysAgo = [...Array(21)].map((_v, idx: number) => {
```

Comment:
can we use some defined variables for our retention length?
```js
const RETENTION_DAYS = 21
```

Reply:
great idea!